azure-pipelines.yaml

```
@@ -10,6 +10,9 @@ parameters:
     displayName: Component to Run
     type: object
     default:
+      - deployment: 'backend'
```

how about this? to be a bit clearer?
```suggestion
      - deployment: 'backend-app-gateway'
```